docs/user/troubleshooting.md

```diff
@@ -52,7 +52,9 @@ If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
 ### Unable to SSH into Master Nodes
 
-In order to SSH into the master nodes, it is necessary to include an administrator's SSH key during the installation. If SSH authentication is failing, ensure that the proper SSH key is being used.
+In order to SSH into the master nodes as user `core`, it is necessary to include an administrator's SSH key during the installation. When asked by the installation wizzard for the ssh-key, make sure you define the absolute path to your key. This key will be added to the core user's authorized_keys file. If installed in aws you will not see this key attached to your instance as a keypair in the aws console. 
+
+If SSH authentication is failing, ensure that the proper SSH key is being used.
```

Comment:
Could we add something like: "Verify your SSH Key by using `oc get configmap -n kube-system cluster-config-v1 -o yaml` etc...? Because I don't think it's clear for people how to verify the key without restarting the install.

Reply:
I think we should still add something like this to the documentation.

Reply:
With the `oc get configmap ...` and associated lead-in above, I think we can drop this line.